AI_Social_Platform.Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace AI_Social_Platform.Extensions
+{
+    using System.Security.Claims;
+
+    public static class ClaimsPrincipalExtensions
```

Comment:
Why do you have this file twice - with diff project names? @antoniliev31 
Please remove one, if this is error.